addon/-private/context.js

```diff
@@ -17,6 +19,11 @@
  * @return {PageObject} - the page object
  */
 export function render(template) {
+  deprecate('PageObject.render() is deprecated. Please use "htmlbars-inline-precompile" instead.', false, {
```

Comment:
Can we provide a link here to some documentation of what it means to use `htmlbars-inline-precompile`?

Reply:
Yeah, seems like the message wants to be improved.

I've been thinking about providing an [`url`](https://github.com/emberjs/ember.js/blob/bdc2c3e3a15d13d8bb1951d44b1a28ee1c6a36cb/packages/ember-debug/lib/deprecate.js#L155) deprecation option pointing to some deprecation specific page or section. There we can explain a migration path. 

The question is what this page/section should be? I can see 2 options:
 - deprecations page on the site
 - separate github issue per each deprecation

I personally like the second option cause it doesn't require to restructure the site. Also separate issues give us an ability to receive feedback from the users in case of related questions. What do you think?

Reply:
I think an issue per deprecation showing before/after examples would be perfect.